docs/standardisation-scope.adoc

```diff
@@ -0,0 +1,199 @@
+= Standardisation scope of Cypher
+
+One of the challenges faced by this effort to standardise Cypher has been to choose the set of language features to standardise.
+Cypher is not a new language, but has several years of history, which includes some historical artifacts left in the feature set.
+This is generally not because there hasn't been good replacements, but because Cypher's role has been to support a specific product (Neo4j).
```

Comment:
"This is generally not because there hasn't been good replacements, but because Cypher's role has been to support a specific product (Neo4j)." 
-->
"This is not because there haven't been good replacements, but because Cypher's role to date has been to support a specific product (Neo4j)."